extmod/moducryptolib.c

```diff
@@ -356,9 +361,190 @@ STATIC const mp_obj_type_t ucryptolib_aes_type = {
     .locals_dict = (void *)&ucryptolib_aes_locals_dict,
 };
 
+#if MICROPY_PY_UCRYPTOLIB_GCM
+
+#if MICROPY_SSL_AXTLS
+#warning GCM is not currentl implementing with axTLS
```

Comment:
I'll fix the typo. The word order is deliberate; since it might be implemented in the future the _not_ is more tightly bound to the currency than the implementation.